MailSender.lib/Service/DBMailSenderService.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.EntityFrameworkCore;
+using MailSender.lib.Models;
+using MailSender.lib.Data;
+
+namespace MailSender.lib.Service
+{
+    public class DBMailSenderService
+    {
+        public static IConfiguration Configuration { get; } = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json", false, true)
+            .Build();
+        public static string ConnectionString => Configuration.GetConnectionString("Default");
+        private DBMailSender db = new DBMailSender(new DbContextOptionsBuilder<DBMailSender>()
+            .UseSqlServer(ConnectionString).Options);
+        public DBMailSenderService()
+        {
+            db.Database.EnsureCreated();
+        }
+        public IEnumerable<SmtpAccount> DbGetAccounts()
+        {
+            return db.SmtpAccounts.ToArray();
```

Comment:
```suggestion
            return db.SmtpAccounts;
```

Если объектов в БД будет оооочень много, то в лучшем случае программа зависнет пока все они из БД не скопируются к Вам в память, а в худшем (наиболее вероятном) случае Вы получите OutOfMemoryException. А так мы возвращаем объект к которому можно сделать запрос и получить только интересующие нас данные, а не вообще все что есть в БД.